pom.xml

```diff
@@ -3,58 +3,40 @@
   <parent>
     <groupId>org.jenkins-ci.plugins</groupId>
     <artifactId>plugin</artifactId>
-    <version>2.37</version>
+    <version>3.2</version>
     <relativePath />
   </parent>
 
   <licenses>
     <license>
       <name>The MIT License (MIT)</name>
-      <url>http://opensource.org/licenses/MIT</url>
+      <url>https://opensource.org/licenses/MIT</url>
       <distribution>repo</distribution>
     </license>
   </licenses>
 
   <artifactId>git</artifactId>
-  <version>3.7.1-SNAPSHOT</version>
+  <version>4.0.0-SNAPSHOT</version>
   <packaging>hpi</packaging>
   <name>Jenkins Git plugin</name>
   <description>Integrates Jenkins with GIT SCM</description>
-  <url>http://wiki.jenkins-ci.org/display/JENKINS/Git+Plugin</url>
+  <url>https://wiki.jenkins.io/display/JENKINS/Git+Plugin</url>
   <inceptionYear>2007</inceptionYear>
 
   <properties>
-    <jenkins.version>1.625.3</jenkins.version>
-    <java.level>7</java.level>
+    <jenkins.version>2.60.3</jenkins.version>
+    <java.level>8</java.level>
     <no-test-jar>false</no-test-jar>
     <concurrency>1C</concurrency>
-    <findbugs.failOnError>false</findbugs.failOnError>
-    <workflow.version>1.14.2</workflow.version>
+    <findbugs.failOnError>true</findbugs.failOnError>
+    <maven-project-info-reports-plugin.version>2.9</maven-project-info-reports-plugin.version>
+    <workflow.version>2.4</workflow.version>
```

Comment:
There is no such thing as a “Workflow version” (or “Pipeline version”) after 1.13. As of 2.x, all plugin versions are independent. Anyway this property seems to be unused, so why not just delete it?